### ISSUE TITLE
Image sizes

### ISSUE THREAD

**Issue by mikeyrayvon** [interglobalvision/globie-wordpress-starter-pack#87]
Its important to set an image size to its widest column. Sometimes this isn't the desktop width column, like in `item-m-6 item-l-3`. Perhaps we should have a naming convention like `col-m-6-portrait-crop`?

**Comment by varffs** [comment]
yes a naming convention is a good idea. probably doesnt need the item so:

`grid-class` - `orientation` - `crop suffix if cropped`

?

**Comment by mikeyrayvon** [comment]
`aspect ratio` somewhere in there perhaps? or is that `orientation`
